python/kfserving/kfserving/test_storage.py

```diff
@@ -34,3 +44,34 @@ def test_no_prefix_local_path():
     relative_path = '.'
     assert kfserving.Storage.download(abs_path) == abs_path
     assert kfserving.Storage.download(relative_path) == relative_path
+
+
+def test_public_gcs():
+    gcs_path = 'gs://kfserving-samples/models/tensorflow/flowers'
+    assert kfserving.Storage.download(gcs_path)
+
+
+def test_private_gcs():
+    if GOOGLE_APPLICATION_CREDENTIALS:
```

Comment:
Since I don't have access to kfserving's testing infrastructure, this test won't run unless someone set the global value `GOOGLE_APPLICATION_CREDENTIALS`. 

Do the kfserving CI/CD expose `GOOGLE_APPLICATION_CREDENTIALS` and S3 credentials in the testing environment? If so, I can update these tests accordingly.

Reply:
here we are doing unit tests not integration tests, so you can set the env in the tests.

Reply:
I updated the unit test to run it when the environment variables are available.